### PR TITLE
Twilio Branding + CLA Policy Update

### PR DESCRIPTION
**Pull request by thinkingserious**
This PR updates the Twilio SendGrid branding and implements a new CLA policy.

### CODE REVIEW

File: .github/PULL_REQUEST_TEMPLATE [hunk 1]
```diff
@@ -1,6 +1,7 @@
 <!--
 We appreciate the effort for this pull request but before that please make sure you read the contribution guidelines given above, then fill out the blanks below.
 
+**All third party contributors acknowledge that any contributions they provide will be made under the same open source license that the open source project is provided under.**
```
Comment (childish-sambino) [inline .github/PULL_REQUEST_TEMPLATE]: nit: [hyphens](https://github.com/twilio/.github/pull/3/files)

File: CONTRIBUTING.md [hunk 2]
```diff
@@ -1,16 +1,16 @@
-Hello! Thank you for choosing to help contribute to one of the SendGrid open source libraries. There are many ways you can contribute and help is always welcome.  We simply ask that you follow the following contribution policies.
+Hello! Thank you for choosing to help contribute to one of the Twilio SendGrid open source libraries. There are many ways you can contribute and help is always welcome.  We simply ask that you follow the following contribution policies.
 
-- [CLAs and CCLAs](#clas-and-cclas)
+**All third party contributors acknowledge that any contributions they provide will be made under the same open source license that the open source project is provided under.**
```
Comment (childish-sambino) [inline CONTRIBUTING.md]: Same as above.

File: README.md [hunk 3]
```diff
@@ -10,11 +10,11 @@
...
 
-This library represents the beginning of a new path for SendGrid. We want this library to be community driven and SendGrid led. We need your help to realize this goal. To help make sure we are building the right things in the right order, we ask that you create [issues](https://github.com/sendgrid/sendgrid-ruby/issues) and [pull requests](https://github.com/sendgrid/sendgrid-ruby/blob/master/CONTRIBUTING.md) or simply upvote or comment on existing issues or pull requests.
+This library represents the beginning of a new path for SendGrid. We want this library to be community driven and Twilio SendGrid led. We need your help to realize this goal. To help make sure we are building the right things in the right order, we ask that you create [issues](https://github.com/sendgrid/sendgrid-ruby/issues) and [pull requests](https://github.com/sendgrid/sendgrid-ruby/blob/master/CONTRIBUTING.md) or simply upvote or comment on existing issues or pull requests.
```
Comment (childish-sambino) [inline README.md]: *new Path for Twilio SendGrid*?

File: TROUBLESHOOTING.md [hunk 4]
```diff
@@ -1,20 +1,21 @@
...
-* [Continue Using v2](#v2)
-* [Testing v3 /mail/send Calls Directly](#testing)
-* [Error Messages](#error)
-* [Versioning](#versioning)
-* [Environment Variables and Your SendGrid API Key](#environment)
-* [Using the Package Manager](#package-manager)
-* [Rails Specifics](#rails-specifics)
-* [Ruby Versions](#ruby-versions)
-* [Viewing the Request Body](#request-body)
+- [Table of Contents](#table-of-contents)
```
Comment (childish-sambino) [inline TROUBLESHOOTING.md]: Prefer hyphen to asterisk?

File: USE_CASES.md [hunk 5]
```diff
@@ -85,7 +99,7 @@ data = JSON.parse('{
       ],
       "dynamic_template_data": {
         "subject": "Testing Templates",
-	"name": "Example User",
+	      "name": "Example User",
```
Comment (childish-sambino) [inline USE_CASES.md]: Looks odd.

File: USE_CASES.md [hunk 6]
```diff
@@ -2,10 +2,24 @@ This documentation provides examples for specific use cases. Please [open an iss
 
 # Table of Contents
 
-* [Transactional Templates](#transactional-templates)
-* [Legacy Templates](#legacy-templates)
-* [How to Setup a Domain Authentication](#domain-authentication)
-* [How to View Email Statistics](#email-statistics)
+- [Table of Contents](#table-of-contents)
```
Comment (childish-sambino) [inline USE_CASES.md]: Self-linking ToC?

File: test/sendgrid/helpers/mail/test_mail.rb [hunk 7]
```diff
@@ -11,25 +11,25 @@ def setup
...
+    mail.subject = "Hello World from the Twilio SendGrid Ruby Library"
     personalization = Personalization.new
     personalization.add_to(Email.new(email: "test@example.com", name: "Example User"))
     personalization.add_to(Email.new(email: "test@example.com", name: "Example User"))
     personalization.add_cc(Email.new(email: "test@example.com", name: "Example User"))
     personalization.add_cc(Email.new(email: "test@example.com", name: "Example User"))
     personalization.add_bcc(Email.new(email: "test@example.com", name: "Example User"))
     personalization.add_bcc(Email.new(email: "test@example.com", name: "Example User"))
-    personalization.subject = "Hello World from the Personalized SendGrid Python Library"
+    personalization.subject = "Hello World from the Personalized Twilio SendGrid Python Library"
```
Comment (childish-sambino) [inline test/sendgrid/helpers/mail/test_mail.rb]: Not Python.

File: test/sendgrid/helpers/mail/test_mail.rb [hunk 8]
```diff
@@ -46,7 +46,7 @@ def test_kitchen_sink
     personalization2.add_cc(Email.new(email: "test@example.com", name: "Example User"))
     personalization2.add_bcc(Email.new(email: "test@example.com", name: "Example User"))
     personalization2.add_bcc(Email.new(email: "test@example.com", name: "Example User"))
-    personalization2.subject = "Hello World from the Personalized SendGrid Python Library"
+    personalization2.subject = "Hello World from the Personalized Twilio SendGrid Python Library"
```
Comment (childish-sambino) [inline test/sendgrid/helpers/mail/test_mail.rb]: Not Python.

File: test/sendgrid/helpers/mail/test_mail.rb [hunk 9]
```diff
@@ -115,7 +115,7 @@ def test_kitchen_sink
...
...Test":"True"},"send_at":1443636843,"subject":"Hello World from the Personalized Twilio SendGrid Python Library","substitutions":{"%city%":"Denver","%name%":"Example User"},"to":[{"email":"test@example.com","name":"Example User"},{"email":"test@example.com","name":"Example User"}]}],"reply_to":{"email":"test@example.com"},"sections":{"%section1%":"Substitution Text for Section 1","%section2%":"Substitution Text for Section 2"},"send_at":1443636842,"subject":"Hello World from the Twilio SendGrid Ruby Library","template_id":"13b8f94f-bcae-4ec6-b752-70d6cb59f932","tracking_settings":{"click_tracking":{"enable":false,"enable_text":false},"ganalytics":{"enable":true,"utm_campaign":"some campaign","utm_content":"some content","utm_medium":"some medium","utm_source":"some source","utm_term":"some term"},"open_tracking":{"enable":true,"substitution_tag":"Optional tag to replace with the open image in the body of the message"},"subscription_tracking":{"enable":true,"html":"html to insert into the text/html portion of the message","substitution_tag":"Optional tag to replace with the open image in the body of the message","text":"text to insert into the text/plain portion of the message"}}}'))
```
Comment (childish-sambino) [inline test/sendgrid/helpers/mail/test_mail.rb]: Still not Python.

File: .github/PULL_REQUEST_TEMPLATE (hunk 1 above)
Comment (thinkingserious) [reply .github/PULL_REQUEST_TEMPLATE]: Updated.

File: CONTRIBUTING.md (hunk 2 above)
Comment (thinkingserious) [reply CONTRIBUTING.md]: Updated

File: README.md (hunk 3 above)
Comment (thinkingserious) [reply README.md]: Correct, good catch!

File: TROUBLESHOOTING.md (hunk 4 above)
Comment (thinkingserious) [reply TROUBLESHOOTING.md]: Ah, looks like my Markdown TOC script was misconfigured.

File: USE_CASES.md (hunk 6 above)
Comment (thinkingserious) [reply USE_CASES.md]: Markdown TOC glitch, fixed. Thanks!

File: test/sendgrid/helpers/mail/test_mail.rb (hunk 7 above)
Comment (thinkingserious) [reply test/sendgrid/helpers/mail/test_mail.rb]: Fixed.

File: test/sendgrid/helpers/mail/test_mail.rb (hunk 8 above)
Comment (thinkingserious) [reply test/sendgrid/helpers/mail/test_mail.rb]: Fixed.

File: test/sendgrid/helpers/mail/test_mail.rb (hunk 9 above)
Comment (thinkingserious) [reply test/sendgrid/helpers/mail/test_mail.rb]: Fixed.

File: .github/PULL_REQUEST_TEMPLATE [hunk 10]
```diff
@@ -1,6 +1,7 @@
 <!--
 We appreciate the effort for this pull request but before that please make sure you read the contribution guidelines given above, then fill out the blanks below.
 
+**All third-party contributors acknowledge that any contributions they provide will be made under the same open source license that the open source project is provided under.**
```
[the remark this answers is not in the document]
Comment (thinkingserious) [reply .github/PULL_REQUEST_TEMPLATE]: Updated.

**Comment by thinkingserious** [comment]
Hello @thinkingserious,<br /><br/>Thanks again for the PR!<br />                                              <br />We appreciate your contribution and look forward to                                              continued collaboration. Thanks!                                             <br /><br />Team SendGrid DX

**Pull request merged**